docs/source/command_line.md

```diff
@@ -0,0 +1,194 @@
+# Command Line Operation
+
+:::{index} Command Line Operation
+:::
+## Running Arelle on the Command Line:
+
+
+| Method or System                     | Command                                                           |
+|--------------------------------------|-------------------------------------------------------------------|
+| Python source code                   | python arelleCmdLine.py *arguments*                               |
+| Windows x64 or 32 bit                | "c:\Program Files\arelleCmdLine.exe" *arguments*                  |
+| MacOS App                            | /Applications/Arelle.app/contents/MacOS/arelleCmdLine *arguments* |
+| Linux/Unix                           | ./arelleCmdLine *arguments*                                       |
+
+## Arguments
+
+### General Arguments
+
+| Flag             | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| -a *or* \--about | Prints build, copyright, and license information then exits |
+| -h *or* \--help  | Prints help message and exits                               |
+| \--diagnostics   | Prints diagnostic information                               |
```

Comment:
Thanks to smart quotes, sphinx is replacing all of these arguments that begin with two hyphens with an em dash. Two backslashes (not one) will prevent this.

```suggestion
| \\--diagnostics   | Prints diagnostic information                               |
```

There are also em dashes in the actual markdown for some of the arguments. It would be good to do a search and replace to make all of those hyphens. If you copy and paste the em dash version of an argument Arelle doesn't recognize it which is likely to makes users all kinds of confused.